deploy/templates/clusterrole.yaml

```diff
@@ -53,6 +53,20 @@ rules:
   - patch
   - update
   - watch
+{{- if .Values.workloadIdentity.enabled }}
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts/token
```

Comment:
are these new permissions required only when `disableGlobalServiceAccount` is true?

Reply:
No, when `disableGlobalServiceAccount ` is false, customers can still use namespaced service account, as long as they are allowed to passing their own service account, we need these role.

Reply:
make sense